src/serialization/Data.ts

```diff
@@ -124,10 +124,20 @@ export interface DataOutput {
 
     writeShortArray(shorts: number[]): void;
 
+    /** @deprecated since version 4.1.
```

Comment:
Since version should be 4.2, we already released 4.1. Also, it would be good if we can also link the suggested method so that users can easily navigate to it while reading the documentation.

Can you also update the other places?